internal/cmd/stack/environment.go

```diff
@@ -30,6 +31,51 @@ type ConfigInput struct {
 	WriteOnly graphql.Boolean `json:"writeOnly"`
 }
 
+type configElement struct {
+	ID        string     `graphql:"id" json:"id,omitempty"`
+	Checksum  string     `graphql:"checksum" json:"checksum,omitempty"`
+	CreatedAt int64      `graphql:"createdAt" json:"createdAt,omitempty"`
+	Runtime   bool       `graphql:"runtime" json:"runtime,omitempty"`
+	Type      ConfigType `graphql:"type" json:"type,omitempty"`
+	Value     *string    `graphql:"value" json:"value,omitempty"`
+	WriteOnly bool       `graphql:"writeOnly" json:"writeOnly,omitempty"`
+	FileMode  *string    `graphql:"fileMode" json:"fileMode,omitempty"`
+}
+
+type listEnvElementOutput struct {
+	Name     string  `json:"name"`
+	Type     string  `json:"type"`
+	Value    *string `json:"value"`
+	FileMode *string `json:"fileMode"`
+	// Context specifies the name of the context
+	Context *string `json:"context"`
+}
+
+type attachedContexts struct {
+	ID          *string         `graphql:"id" json:"id,omitempty"`
+	ContextName *string         `graphql:"contextName" json:"contextName,omitempty"`
+	Config      []configElement `graphql:"config" json:"config,omitempty"`
+}
+
+type runtimeConfig struct {
+	Context *struct {
+		ID          string `graphql:"id" json:"id,omitempty"`
+		ContextName string `graphql:"contextName" json:"contextName,omitempty"`
+	} `graphql:"context" json:"context"`
+	Element configElement `graphql:"element" json:"element,omitempty"`
+}
+
+type listEnvQuery struct {
+	Stack struct {
+		// The attachedContext field may not be needed here. It's just kept for reference and to keep the
+		// implementation consistent with the frontend.
+		AttachedContexts []attachedContexts `graphql:"attachedContexts" json:"attachedContexts"`
```

Comment:
Personally I'd just remove this if it isn't used. I just left it in the example query in case I was wrong about it not being required. Apologies if this caused confusion.

Reply:
Okay, I've removed this part.